lib/vagrant-service-manager/command.rb

```diff
@@ -86,8 +87,20 @@ def check_if_a_service_is_running?(service)
 
       def print_all_provider_info
         @env.ui.info I18n.t('servicemanager.commands.env.nil')
-        self.execute_docker_info
-        self.execute_openshift_info
+
+        running_services = []
+        SUPPORTED_SERVICES.each do |service|
```

Comment:
>  Would agree if we want to keep service name as that of systemd name.

I kept the service name in the command output as that of respective systemd service name.

>Writing @env.ui.info once is more salable even if we trades off it with one more variable.

Updated the PR accordingly.